src/Components/button/button.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import './button.css'
+function Button (props) {
```

Comment:
extract the props like this
```suggestion
function Button ({ text, onClick }) {
```